include/zephyr/drivers/lora.h

```diff
@@ -46,13 +46,36 @@ enum lora_coding_rate {
 };
 
 struct lora_modem_config {
+	/* Frequency in Hz to use for transceiving */
 	uint32_t frequency;
+
+	/* The bandwidth to use for transceiving */
 	enum lora_signal_bandwidth bandwidth;
+
+	/* The data-rate to use for transceiving */
 	enum lora_datarate datarate;
+
+	/* The coding rate to use for transceiving */
 	enum lora_coding_rate coding_rate;
+
+	/* Length of the preamble */
 	uint16_t preamble_len;
+
+	/* TX-power in dBm  to use for transmission */
 	int8_t tx_power;
+
+	/* Set to true for transmission, false for receiving */
 	bool tx;
+
+	/* Invert the IQ. Usually can be set to false, but should be set to true for sending a downlink. */
+	bool iq_inverted;
+
+	/* Sets the sync-byte to use: 
+	 * true for using the public network sync byte
+	 * false for using the private network sync byte
+	 * WARNING: Only set to true when sending LoRaWAN packets
+	 */
+	bool public_network;
```

Comment:
These two new parameters should also be explicitly set in the 2 LoRa samples (`samples/drivers/lora`) in a third commit.